cumulusci.yml

```diff
@@ -880,18 +880,18 @@ flows:
                         Callable apiClass = (Callable)t.newInstance();
                         apiClass.call('RD2.ExecuteDataMigration', params);
             6:
-                task: batch_apex_wait
-                # Wait until after the data migration job has completed
-                options:
-                    class_name: RD2_DataMigration_BATCH
-            7:
                 task: execute_anon
                 # Update enhanced Recurring Donation enablement page state
                 options:
                     apex: >
                         npe03__Recurring_Donations_Settings__c rdSettings = npe03__Recurring_Donations_Settings__c.getOrgDefaults();
                         rdSettings.%%%NAMESPACE%%%RecurringDonations2EnablementState__c = '{"isReady":true,"isMigrationEnabled":true,"isMetaLaunched":true,"isMetaConfirmed":true,"isEnabled":true,"isDryRun2":false,"isConfirmed":true,"dryRunLimit":7}';
                         upsert rdSettings;
+            7:
```

Comment:
Looking back at that slack thread, it may be that you need to move step 6 (update settings) to be before step 5 (execute data migration).

Reply:
It has been fixed, thank you!